recipe/meta.yaml

```diff
@@ -30,10 +30,15 @@ requirements:
     - pip
     - pybind11
     - python
+    - cuda-cudart-dev  # [(cuda_compiler_version or "").startswith("12")]
+    - libcublas-dev    # [(cuda_compiler_version or "").startswith("12")]
+    - libcusparse-dev  # [(cuda_compiler_version or "").startswith("12")]
```

Comment:
@vincentmr I think this PR is ready. I had to apply the migrator manually because the bot didn't do it for us (). The migrator is enabling CUDA 12 builds, which cuQuantum already supports since 23.03. The CUDA 12 support is a major update done by NVIDIA/conda-forge. We changed the package layout to better serve the community (no more all-in-one `cudatoolkit` + Docker image hacks, see https://github.com/conda-forge/conda-forge.github.io/issues/1963). Feel free to ask me any question you might have 🙂

In the meanwhile, would you confirm that lightning-gpu needs cuBLAS/cuSPARSE? I had to add them to avoid CMake errors and enable the build.

Reply:
Hi @leofang , thanks very much for the spadework. We planned moving to CUDA-12 and deprecate CUDA-11 builds later this year. I'll revise it, but it looks good :)

> In the meanwhile, would you confirm that lightning-gpu needs cuBLAS/cuSPARSE? I had to add them to avoid CMake errors and enable the build.

Yes, both required.

Reply:
Thanks, Vincent. cuQuantum will likely support both CUDA versions indefinitely, until the next CUDA major release is out.